src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,6 +127,27 @@ public void setMostRecentGroupTaskDisplay(String string) {
         }
     }
 
+    @Override
+    public Task getMostRecentTaskDisplay() {
+        return userPrefs.getMostRecentTaskDisplay();
+    }
+
+    @Override
+    public void setMostRecentTaskDisplay() {
```

Comment:
can you explain the purpose for this set of code?

Reply:
When the new task is added the GUI is set to jump to the Task Panel. There is a filter which allows it to show the specific task that is added only, and not all the other tasks. This is the task which is saved into the mostRecentTaskDisplay, and into the system memory. Its function is so that when a user decides to exit the application upon adding a specific task to all groups, when the user next boots up the app again the Task Panel can remain as it was because that most recent task was saved. So it allows the Task Panel to filter through all the tasks using saved memory, displaying the one that is needed.